processing/data_collection/gazette/spiders/ce_associacao_municipios.py

```diff
@@ -0,0 +1,14 @@
+from datetime import date
+
+from gazette.spiders.base import DiarioMunicipalSigpubGazetteSpider
+
+
+class CeAssociacaoMunicipiosSpider(DiarioMunicipalSigpubGazetteSpider):
+    name = "ce_associacao_municipios"
+    TERRITORY_ID = "2300000"
+
+    CALENDAR_URL = "http://www.diariomunicipal.com.br/aprece"
```

Comment:
As mentioned before, we can cleanup this as it's all the same. The only required bit is `aprece`.